Week-4/day10/jsMaps/script.js

```diff
@@ -0,0 +1,11 @@
+const calcWordFrequencies = () => {
+  let userInput = prompt("Enter a sentence");
+  const wordArr = userInput.split(" ");
```

Comment:
Consider using a regular expression to handle splitting by spaces more robustly, to account for multiple consecutive spaces.